### ISSUE TITLE
Dynamo 2.0.3. does not open in Revit

### ISSUE THREAD

**Issue by DanimoSite** [DynamoDS/Dynamo#9818]
## Dynamo version

Dynamo 2.0.3.8810

## Operating system

Windows 10

## What did you do? 
Try and open Dynamo 2.0.3 in Revit

## What did you expect to see?

Dynamo to open

## What did you see instead?

The following error message...

![image](https://user-images.githubusercontent.com/37482336/60246374-77081900-98b6-11e9-8f52-630fad52d065.png)

## Additional Info
This error is happening on a colleagues build. He also had Revit 2020/Dynamo 2.1 installed first. Not sure if this is related, but wondered if this has something to do with the libg folders have different folder names to my own build of Dynamo 2.0.3. Let me know if you need any further info.

**Comment by mjkkirschner** [comment]
Hi @DanimoSite - 
will need some more info:

1. did your colleague install any pre release dynamo builds? like 2.1 from the builds page? - if so get rid of that core!
2. can you show their control panel of dynamo installations?
3. the libG folders should have the same names if you have 2.0.3 installed as well - can you screenshot it? Should have the form LibG_x_y_z if they are using 2.0.3 - LiBG_x - is an older form.
4. The error implies Dynamo can't find the right libG folder for the revit version.

**Comment by DanimoSite** [comment]
Thanks @mjkkirschner,

I did ask my colleague already to check the libg folders and they are different from my own. I wasn't sure if this was to do with any Dynamo 2.1 API changes.

His Dynamo 2 Core directory...
![image](https://user-images.githubusercontent.com/37482336/60339545-f70aad80-99a0-11e9-822d-c17d3134f812.png)

My Dynamo Core directory...
![image](https://user-images.githubusercontent.com/37482336/60339619-25888880-99a1-11e9-9ebc-d57439da1ee4.png)

I don't think he has installed any builds that are from the Dynamo Builds page (apart from the latest stable build of 2.0.3.). Dynamo 2.1 came with Revit 2020.

The Dynamo installs he currently has is the latest 1.3.4, 2.0.3 stable builds and whatever build came with Revit 2020. I can get full build numbers if this helps.

I will check about Advanced Steel, I did get him to temporarily remove addins for Revit to check for any conflicts, but this didn't fix the issue.
